cloud/aws/modules/setup/secrets.tf

```diff
@@ -0,0 +1,90 @@
+# Firstly create a random generated password to use in secrets.
+resource "random_password" "postgres_password" {
+  length           = 16
+  special          = true
+  override_special = "_%@"
+}
+
+# Creating a AWS secret for postgres_password
+resource "aws_secretsmanager_secret" "postgres_password_secret" {
+  name = "postgres_password"
+}
+
+# Creating a AWS secret versions fro postgres_password
+resource "aws_secretsmanager_secret_version" "postgres_password_secret_version" {
+  secret_id     = aws_secretsmanager_secret.postgres_password_secret.id
+  secret_string = random_password.postgres_password.result
+}
+
+# Firstly create a random generated password to use in secrets.
+resource "random_password" "app_secret_key" {
+  length           = 16
+  special          = true
+  override_special = "_%@"
+}
+
+# Creating a AWS secret for app_secret_key
+resource "aws_secretsmanager_secret" "app_secret_key_secret" {
+  name = "app_secret_key"
+}
+
+# Creating a AWS secret versions fro app_secret_key
+resource "aws_secretsmanager_secret_version" "app_secret_key_secret_version" {
+  secret_id     = aws_secretsmanager_secret.app_secret_key_secret.id
+  secret_string = random_password.app_secret_key.result
+}
+
+# Creating a AWS secret for adfs_secret
+resource "aws_secretsmanager_secret" "adfs_secret_secret" {
+  name = "adfs_secret"
+}
+
+# Creating a AWS secret versions fro adfs_secret
+resource "aws_secretsmanager_secret_version" "adfs_secret_secret_version" {
+  secret_id     = aws_secretsmanager_secret.adfs_secret_secret.id
+  secret_string = " "
+}
+
+# Creating a AWS secret for aws_secret_access_token
+resource "aws_secretsmanager_secret" "aws_secret_access_token_secret" {
```

Comment:
where do you we use these? I would imagine it will be handled by AIM roles

Reply:
i think the app expects this to be passed in for the time being so it can send emails, but ya maybe this isn't necessary